packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -81,4 +81,12 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
         break;
     }
   }
+
+  private _onFocus() {
```

Comment:
Not for eventgroup event callbacks, which uses native eventing and auto bind for you. Id recommend using react eventing over native unless necessary. I wrote an explanation in bestpractices.md.